meeting-notes/2018/2018-09-17--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,173 @@
+  - Revert libp2p records (IPNS) being signed for interop [js-ipfs#1570](https://github.com/ipfs/js-ipfs/pull/1570)
+  - Started interop tests for `IPNS over Pubsub`
+  - Started interop tests for `DHT`
+- Blocked:
+  - `IPNS Interop`
+    - [js-ipfs#1570](https://github.com/ipfs/js-ipfs/pull/1570) - Needs review and release
+    - [interop#26](https://github.com/ipfs/interop/pull/26) - Needs [js-ipfs#1570](https://github.com/ipfs/js-ipfs/pull/1570) and code review
+  - `IPNS over Pubsub`
+    - [interface-ipfs-core#361](https://github.com/ipfs/interface-ipfs-core/pull/361)
+    - [js-ipfs-api#846](https://github.com/ipfs/js-ipfs-api/pull/846)
+    - [js-ipfs#1559](https://github.com/ipfs/js-ipfs/pull/1559)
+    - All need review/release and are dependencies of each other from top to bottom
+- Next:
+  - Fix all the feedback bavailable for my blocked PRs
+  - `DHT Interop` tests
+    - WIP [interop#36](https://github.com/ipfs/interop/pull/36)
+    - Call with Cole (@bigs)
+  - `IPNS over Pubsub Interop` tests
+
+@alanshaw
+- Done:
+    - Released `js-ipfs` 0.32! https://ipfs.io/blog/43-js-ipfs-0-32/
+    - Created `jsipfs cid` tool [#1560](https://github.com/ipfs/js-ipfs/pull/1560)
+    - Worked on `--cid-base` option [#1552](https://github.com/ipfs/js-ipfs/pull/1552)
+- Blocked:
+    - Jenkins npm errors frequent [#51](https://github.com/ipfs/testing/issues/51)
+    - Jenkins windows workers out of space [#135](https://github.com/ipfs/testing/issues/135)
+- Next:
+    - Finish `--cid-base` option
+    - Create interop tests for block store get by CIDv0/1
+    - Implement block store get by CIDv0/1
+    
+@hugomrdias
+- Done
+  - ipfs add experimental [https://github.com/ipfs/js-ipfs/pull/1540](https://github.com/ipfs/js-ipfs/pull/1540)
+  - ipfs-api add experimental [https://github.com/ipfs/js-ipfs-api/pull/851](https://github.com/ipfs/js-ipfs-api/pull/851)
+- Blocked
+  - jsdoc rule fix [#265](https://github.com/ipfs/aegir/pull/265)
+  - improve ctl docs [#pull/294](https://github.com/ipfs/js-ipfsd-ctl/pull/294)
+  - Doesn't work inside a service-worker on Firefox [848#issuecomment-419887279](https://github.com/ipfs/js-ipfs-api/issues/848#issuecomment-419887279)
+  - stream-http improvements [stream-http/tree/fix/body-handling](https://github.com/hugomrdias/stream-http/tree/fix/body-handling)
+- Next
+  - release ipfsd-ctl with pregen peer-id , docs, updated deps etc (try to include fix for node ≥ 10.9)
+  - help out with examples to embed in js.ipfs.io
+  - OKR stuff [https://github.com/ipfs/js-ipfs/pull/1566#event-1847679907](https://github.com/ipfs/js-ipfs/pull/1566#event-1847679907)
+
+
+@travisperson
+ - Done:
+   - js ipfs / browser plugins merged (https://github.com/ipfs/iptb/pull/69)
+ - Blocked:
+   - iptb interopt tests
+ - Next:
+   - iptb plugin documentation
+
+@vmx
+ - Done:
+   - GraphSync (I call it Transsub) first public prototype: https://github.com/vmx/js-transsub
```

Comment:
@vmx is there a spec or architecture doc? Also, please record a demo 🙏🏽

Reply:
Architecture doc is at https://github.com/ipld/specs/pull/66